docs/sources/clients/promtail/configuration.md

```diff
@@ -1622,6 +1626,116 @@ users with thousands of services it can be more efficient to use the Consul API
+# resources.
+# The available filters are listed in the upstream documentation:
+# Containers: https://docs.docker.com/engine/api/v1.41/#operation/ContainerList
+[ filters:
+  [ - name: <string>
+      values: <string>, [...] ]
+]
+
+# The time after which the containers are refreshed.
+[ refresh_interval: <duration> | default = 60s ]
+
+# Authentication information used to authenticate to the Docker daemon.
+# Note that `basic_auth` and `authorization` options are
+# mutually exclusive.
+# password and password_file are mutually exclusive.
+
+# Optional HTTP basic authentication information.
+basic_auth:
+  [ username: <string> ]
+  [ password: <secret> ]
+  [ password_file: <string> ]
+
+# Optional `Authorization` header configuration.
+authorization:
+  # Sets the authentication type.
+  [ type: <string> | default: Bearer ]
+  # Sets the credentials. It is mutually exclusive with
+  # `credentials_file`.
+  [ credentials: <secret> ]
+  # Sets the credentials to the credentials read from the configured file.
+  # It is mutually exclusive with `credentials`.
+  [ credentials_file: <filename> ]
+
+# Optional OAuth 2.0 configuration.
+# Cannot be used at the same time as basic_auth or authorization.
+oauth2:
+  [ <oauth2> ]
+
+# Configure whether HTTP requests follow HTTP 3xx redirects.
+[ follow_redirects: <bool> | default = true ]
+```
+
+Available meta labels:
+
+  * `__meta_docker_container_id`: the id of the container
+  * `__meta_docker_container_name`: the name of the container
+  * `__meta_docker_container_network_mode`: the network mode of the container
+  * `__meta_docker_container_label_<labelname>`: each label of the container
+  * `__meta_docker_container_log_stream`: the log stream type `stdout` or `stderr`
+  * `__meta_docker_network_id`: the ID of the network
+  * `__meta_docker_network_name`: the name of the network
+  * `__meta_docker_network_ingress`: whether the network is ingress
+  * `__meta_docker_network_internal`: whether the network is internal
+  * `__meta_docker_network_label_<labelname>`: each label of the network
+  * `__meta_docker_network_scope`: the scope of the network
+  * `__meta_docker_network_ip`: the IP of the container in this network
+  * `__meta_docker_port_private`: the port on the container
+  * `__meta_docker_port_public`: the external port if a port-mapping exists
+  * `__meta_docker_port_public_ip`: the public IP if a port-mapping exists
+
+These labels can be used during relabelling. For instance, the following configuration scrapes the container named `flog` and removes the leading `/` form the container name.
+
+```yaml
+scrape_configs:
+  - job_name: flog_scrape 
+    docker_sd_configs:
+      - host: unix:///var/run/docker.sock
+        refresh_interval: 5s
+        filters:
+          - name: name
+            values: [flog] 
+    relabel_configs:
+      - source_labels: ['__meta_docker_container_name']
+        regex: '/(.*)'
```

Comment:
Not sure you need regex I think you can use keep action

Reply:
The name is `/flog`. How would I strip the leading `/`? It's just an example.